runtime-testsuite/test/org/antlr/v4/test/runtime/descriptors/ParserErrorsDescriptors.java

```diff
@@ -436,7 +436,7 @@
 
 		/**
 		 grammar T;
-		 a : 'a' ('b'|'z'{<Pass()>})*;
+		 a : 'a' ('b'|'z'{<Pass()>})* EOF ;
```

Comment:
:memo: The original behavior of the test was retained by ensuring the closure loop didn't look all the way to the end of `a`.